android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/RegisterFragment.kt

```diff
@@ -216,12 +218,12 @@ class RegisterFragment : Fragment(), OnSyncListener {
         // Show error message in snackBar message
         // syncJobStatus.exceptions may be null when worker fails; hence the null safety usage
         val hasAuthError =
-          syncJobStatus.exceptions.any {
+          syncJobStatus.exceptions?.any {
             it.exception is HttpException && (it.exception as HttpException).code() == 401
           }
-        Timber.e(syncJobStatus.exceptions.joinToString { it.exception.message ?: "" })
+        Timber.e(syncJobStatus?.exceptions?.joinToString { it.exception.message ?: "" })
```

Comment:
I think we might need to revert these, it appears the properties are already null safe cc @ellykits

Reply:
yes agree and above

Reply:
and below those changes aren't necessary